src/net/sf/memoranda/test/TestHour.java

```diff
@@ -0,0 +1,66 @@
+package net.sf.memoranda.test;
```

Comment:
The filename should probably follow the same convention given in JUnit, so maybe something like HourlyEventTest.java (having Test at the end and a good idea of what it's testing in the title).